app/views/docs/functions-recipes.phtml

```diff
@@ -0,0 +1,6 @@
+<p>
+    Appwrite Functions is all about flexibility. 
+    Behind the simple workflow hides some useful recipes that can help you accomplish your goals faster.
+    Take a look at the following.
+</p>
+[TODO: @matej @luke -> Let's have some simple recipes here for common actions]
```

Comment:
@loks0n @Meldiron

Reply:
For example: Matej suggested these, would be a starting point:
```md
Hmm here is an idea, let me know what you think:

Section 1: My first function

We inform we are going to build currency convertion function
Create new file, index.js
We put some very very simple code into index.js, statically returning 1.13 (because 1€ = 1.13$)
We use git to init, commit, add remote and push
We create function in Console
We exectue function and see response 1.13 (by visiting URL like ghrfu9ewji.functions.appwrite.app)

Section 2: Let's use Payload
We update code in index.js to use req.query.amount. In URL param we recieve how much money you want to convert. Let's say body is 5 (€), we would return 5.63 ($). Still only using static math ($ = € * 1.13)
Push, wait a little, refresh static domain
Play with different params and see resposne number change

Section 3: Installing dependencies
Run npm init --yes to prepare dependency file
Install HTTP client library, npm install axios
Use axios with exchange API to get the rate instead of having it static. FOr example this, it's 100% free without limits: https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest/currencies/eur/usd.json
Push, wait a little, refresh static domain
Play around and see same results, just a bit more precise

Section 4: More routes
Edit index.js to support multipel paths like /eur, /czk, /ft, /rub...
Push, deploy..
Try different endpoints, see they all convert that currency into $


Aaaand we can make this very generic, and have code and command snippets switchable between all runtimes
```

Reply:
done for js

Reply:
done for php dart ruby python